ebpf-io/ebpf.io-website
# feat: add nyt case to case studies page

**This PR:**
Added New York Times case to case studies page

**Steps to test**

- Open the [Case studies page]()
- Confirm that everything looks and works as expected
